.github/workflows/mtags-auto-release.yml

```
@@ -8,11 +8,11 @@ on:
       metals_version:
         description: "Metals Version"
         required: true
-        default: "v0.11.2"
+        default: "v0.11.3"
       metals_ref:
         description: "Tag/branch-name from which run release"
         required: true
-        default: "v0.11.2_mtags_release"
+        default: "v0.11.3"
```

```suggestion
        default: "v0.11.3_mtags_release""
```
This one is on purpose

---

At the moment of release it's fine to have this ref pointing at tag.
I create `*_mtags_release` branch to backport some changes from main only if  cross tests starting fail with newer scala versions or there are issues in workflow